ext/json/json.c

```diff
@@ -92,11 +92,37 @@ static const zend_function_entry json_serializable_interface[] = {
 };
 /* }}} */
 
+/* {{{ PHP_INI_MH
+ */
+static PHP_INI_MH(OnSetJsonPrecision)
+{
+	zend_long i;
+
+	ZEND_ATOL(i, ZSTR_VAL(new_value));
+	if (i >= -1) {
+		JSON_G(precision) = i;
+		return SUCCESS;
+	} else {
+		return FAILURE;
+	}
+}
+/* }}} */
+
+
+/* {{{ PHP_INI
+ */
+PHP_INI_BEGIN()
+STD_PHP_INI_ENTRY("json.precision", "-1",  PHP_INI_ALL, OnSetJsonPrecision, precision, zend_json_globals, json_globals)
+PHP_INI_END()
```

Comment:
I'd actually prefer json precision as it's more flexible. the rfc also contains voting option for that so think it's fine to leave it (it can be removed if it's rejected though).